### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by bravo1goingdark**
added correct syntax for renaming table or column

### CODE REVIEW

File: src/data/roadmaps/sql/content/102-ddl/index.md [hunk 1]
```diff
@@ -35,9 +35,16 @@ Data Definition Language (DDL) is a subset of SQL. Its primary function is to cr
...
 
     ```sql
-    RENAME TABLE old_table_name TO new_table_name;
+    ALTER TABLE table_name
+    RENAME TO new_table_name;
     ```
+    ```sql
+    ALTER TABLE table_name
+    RENAME COLUMN old_column_name TO new_column_name;
+
```
Comment (dansholds) [inline src/data/roadmaps/sql/content/102-ddl/index.md:44]: Please remove this whitespace, apart from that LGTM!

**Review by dansholds — CHANGES_REQUESTED** [review]
Please fix the comment I made!

In future please try to write a more descriptive PR title i.e. "Add this to that"

File: src/data/roadmaps/sql/content/102-ddl/index.md (hunk 1 above)
Comment (kamranahmedse) [inline src/data/roadmaps/sql/content/102-ddl/index.md:44]:
```suggestion
```

File: src/data/roadmaps/sql/content/102-ddl/index.md [hunk 2]
```diff
@@ -35,9 +35,15 @@ Data Definition Language (DDL) is a subset of SQL. Its primary function is to cr
...
     ```sql
-    RENAME TABLE old_table_name TO new_table_name;
+    ALTER TABLE table_name
+    RENAME TO new_table_name;
     ```
+    ```sql
+    ALTER TABLE table_name
+    RENAME COLUMN old_column_name TO new_column_name;
+    ```
+
```
Comment (kamranahmedse) [inline src/data/roadmaps/sql/content/102-ddl/index.md:45]:
```suggestion
```

File: src/data/roadmaps/sql/content/102-ddl/index.md [hunk 3]
```diff
@@ -35,9 +35,14 @@ Data Definition Language (DDL) is a subset of SQL. Its primary function is to cr
 5. `RENAME`: This is used to rename an object in the database.
 
     ```sql
-    RENAME TABLE old_table_name TO new_table_name;
+    ALTER TABLE table_name
+    RENAME TO new_table_name;
+    ```
+    ```sql
```
Comment (kamranahmedse) [inline src/data/roadmaps/sql/content/102-ddl/index.md:41]:
```suggestion
   ; To rename a column
```

File: src/data/roadmaps/sql/content/102-ddl/index.md [hunk 4]
```diff
@@ -35,9 +35,13 @@ Data Definition Language (DDL) is a subset of SQL. Its primary function is to cr
 5. `RENAME`: This is used to rename an object in the database.
 
     ```sql
-    RENAME TABLE old_table_name TO new_table_name;
+    ALTER TABLE table_name
+    RENAME TO new_table_name;
+   ; To rename a column
```
Comment (kamranahmedse) [inline src/data/roadmaps/sql/content/102-ddl/index.md:40]:
```suggestion
  -- To rename a column
```

File: src/data/roadmaps/sql/content/102-ddl/index.md [hunk 5]
```diff
@@ -35,9 +35,13 @@ Data Definition Language (DDL) is a subset of SQL. Its primary function is to cr
 5. `RENAME`: This is used to rename an object in the database.
 
     ```sql
-    RENAME TABLE old_table_name TO new_table_name;
+    ALTER TABLE table_name
+    RENAME TO new_table_name;
+  -- To rename a column
+    ALTER TABLE table_name
+    RENAME COLUMN old_column_name TO new_column_name;
```
Comment (kamranahmedse) [inline src/data/roadmaps/sql/content/102-ddl/index.md:42]:
```suggestion
    -- To rename a table
    ALTER TABLE table_name
    RENAME TO new_table_name;
    
    -- To rename a column
    ALTER TABLE table_name
    RENAME COLUMN old_column_name TO new_column_name;
```

**Pull request merged**